src/linker/specifications/pipeline_2_0_0.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
```

Comment:
Not sure I see the utility of the outer `steps` key, but I don't feel strongly.

Reply:
I don't recall the decision point but that's how all of the pipeline yamls are. I'll leave for now and happy to change in the future.